IWSSL2020.bib

```diff
@@ -143,7 +143,7 @@ @InProceedings{lieberman20
   title =	 {Intrinsic and Extrinsic Motivation in Intelligent
                   Systems},
   author =	 {Lieberman, Henry},
-  pages =	 {62--71},
+  pages =	 {71--79},
```

Comment:
Did this paper get shorter, shouldn't this be 80?

Reply:
Sigh, yes, you are right.  I'll need to make the fixes and try again.